### PR TITLE
Db location

### PR DESCRIPTION
**Pull request by bbelderbos**

### CODE REVIEW

File: src/pybites_search/base.py [hunk 1]
```diff
@@ -9,15 +10,19 @@
...
-CACHE_EXPIRATION_SECONDS = config(
-    "CACHE_EXPIRATION_SECONDS", default=ONE_DAY_IN_SECONDS
-)
 TIMEOUT = 5
 
 console = Console()
 error_console = Console(stderr=True, style="bold red")
 
-requests_cache.install_cache("search_cache", expire_after=CACHE_EXPIRATION_SECONDS)
+HOME_DIR = str(Path.home())
```
Comment (bbelderbos) [inline src/pybites_search/base.py:18]: Interestingly was using TMP first, but that got flagged by flake8 (S108) then went with tempfile.gettempdir() but then the file might not persist. Turns out I had already written in changelog ("document driven design" haha) that we would default to user's home which makes more sense.

File: src/pybites_search/base.py [hunk 2]
```diff
@@ -9,15 +10,19 @@
...
+CACHE_DB_LOCATION = config("CACHE_DB_LOCATION", default=HOME_DIR)
+CACHE_DB_PATH = Path(CACHE_DB_LOCATION) / ".pybites_search_cache.sqlite"
```
Comment (bbelderbos) [inline src/pybites_search/base.py:20]: a dot file so ls does not list it (unless used with -a)

File: CHANGELOG.md [hunk 3]
```diff
@@ -5,6 +5,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.0.12] - 2023-04-21
```
Comment (bbelderbos) [inline CHANGELOG.md:8]: @greywidget I like this changelog format

**Review by greywidget — APPROVED** [review]
Pulled down your db-location branch and had a play.

Home Location working nicely, on windows for me this is `C:\Users\Craig`

On windows, prefacing files/folders with `.` doesn't auto hide them in Windows Explorer although I think some Windows Apps will follow the convention of treating them as hidden.

Look good and works fine, I like it.

**Comment by bbelderbos** [comment]
@greywidget thanks for testing it out!

**Pull request merged**